### PR TITLE
[READY] za hando - a pr about ling claws and punching shit

### PR DESCRIPTION
**Pull request by Hatterhat**
## About The Pull Request
- pugilists literally cannot miss while punching people, for they are required to be punching people
- unarmed attack grammar is now past tense like every other attack
- guerilla gloves and gorilla gloves now have the strip modifiers of their precursors (black gloves and combat gloves, respectively - because apparently...they did have those?)
THE MAJOR SHIT:
- changelings can now mutate Bone Gauntlets
- bone claws/gauntlets let you punch people harder and faster, altclick to punch Even Harder (but slower)
- also the claws/gauntlets have sprites from @Bapldap 
image

slight issue: gauntlets can be hidden because they're just gloves. if there was a flag to make them show over suits that'd be great
## Why It's Good For The Game
punching good
## Changelog
uhh remind me to write one actually
:cl:
add: Changeling bone gauntlets! They punch the shit out of people really good.
tweak: Guerilla gloves and gorilla gloves inherit the strip modifiers of their predecessors, because apparently they had those.
balance: Pugilists now always hit the targeted limb and never miss.
/:cl:

### CODE REVIEW

File: code/modules/antagonists/changeling/powers/mutations.dm [hunk 1]
```diff
@@ -570,3 +574,153 @@
...
+	loudness = 2
+	req_human = 1
+	action_icon = 'icons/mob/actions/actions_changeling.dmi'
+	action_icon_state = "ling_armor"
+	action_background_icon_state = "bg_ling"
+
+	glove_type = /obj/item/clothing/gloves/claws/bone_gauntlets // just punch his head off dude
+	glove_name_simple = "bone gauntlets"
+
+/obj/item/clothing/gloves/claws/bone_gauntlets // should i have just made these a subtype of armwraps?
```
Comment (necromanceranne) [inline code/modules/antagonists/changeling/powers/mutations.dm]: May as well subtype these. Is there any particular reason why you shouldn't? It looks like mostly copypasted from pugilist gloves anyway.

File: code/modules/antagonists/changeling/powers/mutations.dm [hunk 2]
```diff
@@ -570,3 +574,153 @@
...
+			H.dna.species.punchdamagelow += enhancement
+
+/obj/item/clothing/gloves/claws/bone_gauntlets/dropped(mob/user)
+	REMOVE_TRAIT(user, TRAIT_PUGILIST, GLOVE_TRAIT)
+	REMOVE_TRAIT(user, inherited_trait, GLOVE_TRAIT)
+	REMOVE_TRAIT(user, secondary_trait, GLOVE_TRAIT)
+	if(ishuman(user))
+		var/mob/living/carbon/human/H = user
+		H.dna.species.punchdamagehigh -= enhancement
+		H.dna.species.punchdamagelow -= enhancement
```
Comment (necromanceranne) [inline code/modules/antagonists/changeling/powers/mutations.dm]: Was this change mirrored on pugilist gloves, I know someone did it on skyrat and it'd be lovely if that got fixed here on the upstream.

File: code/modules/antagonists/changeling/powers/mutations.dm (hunk 1 above)
Comment (Hatterhat) [reply code/modules/antagonists/changeling/powers/mutations.dm]: y'know what yeah i was debating just making them a pugilist subtype

File: code/modules/antagonists/changeling/powers/mutations.dm (hunk 2 above)
Comment (Hatterhat) [reply code/modules/antagonists/changeling/powers/mutations.dm]: yes

File: code/modules/mob/living/carbon/human/species.dm [hunk 3]
```diff
@@ -1358,6 +1359,7 @@ GLOBAL_LIST_EMPTY(roundstart_race_names)
 				user.do_attack_animation(target, ATTACK_EFFECT_PUNCH)
 
 		var/damage = rand(user.dna.species.punchdamagelow, user.dna.species.punchdamagehigh)
+		var/woundbonus = user.dna.species.woundbonus
```
Comment (necromanceranne) [inline code/modules/mob/living/carbon/human/species.dm]:
```suggestion
		var/woundbonus = user.dna.species.punchwoundbonus
```

Comment (necromanceranne) [reply code/modules/mob/living/carbon/human/species.dm]: Made a nyoops

File: code/modules/antagonists/changeling/powers/mutations.dm (hunk 2 above)
Comment (necromanceranne) [reply code/modules/antagonists/changeling/powers/mutations.dm]: Cool beans

File: code/modules/mob/living/carbon/human/species.dm (hunk 3 above)
Comment (Hatterhat) [reply code/modules/mob/living/carbon/human/species.dm]: fuck

Comment (lolman360) [reply code/modules/mob/living/carbon/human/species.dm]: cattercat nyoops

**Comment by Hatterhat** [comment]
@silicons same deal as the other currently open PR
that being "rerun travis or tell me if/when it's testmerged"

**Pull request merged**